floats/floats.go

```diff
@@ -733,6 +811,48 @@ func Span(dst []float64, l, u float64) []float64 {
 	if n < 2 {
 		panic("floats: destination must have length >1")
 	}
+
```

Comment:
We'll need to add commentary about the NaN and Inf cases.

Reply:
Panic if `l > u`?

Reply:
Same here.